### PR TITLE
12256 Remove unneeded Common config and update V4 support lib

### PR DESCRIPTION
**Pull request by Sergio-Mira**
https://swrvedev.jira.com/browse/SWRVE-12256
https://swrvedev.jira.com/browse/SWRVE-12208

To fix these issues:
https://github.com/Swrve/swrve-android-sdk/issues/126
https://github.com/Swrve/swrve-android-sdk/issues/167

### CODE REVIEW

**Comment by Sergio-Mira** [comment]
@dominicmarmionswrve Could you have a quick look?

**Comment by dominicmarmionswrve** [comment]
There's a jira for one of these https://swrvedev.jira.com/browse/SWRVE-12256
Is there another for the other issue?

File: SwrveSDKCommon/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -1,11 +1,4 @@
...
     package="com.swrve.swrvesharedsdk">
-
-    <application
-        android:allowBackup="true"
-        android:label="@string/app_name"
-        android:supportsRtl="true">
-
-    </application>
-
+    <application android:label="@string/app_name"/>
```
Comment (dominicmarmionswrve) [inline SwrveSDKCommon/src/main/AndroidManifest.xml:3]: 👍

File: gradle.properties [hunk 2]
```diff
@@ -22,5 +22,5 @@ ANDROID_BUILD_MIN_SDK_VERSION=10
 ANDROID_BUILD_TARGET_SDK_VERSION=23
 ANDROID_BUILD_SDK_VERSION=23
 ANDROID_BUILD_TOOLS_VERSION=23.0.2
-ANDROID_DESIGN_TOOLS_VERSION=23.+
+ANDROID_DESIGN_TOOLS_VERSION=23.3.0
```
Comment (dominicmarmionswrve) [inline gradle.properties:25]: Still need to test this out.

File: SwrveSDKCommon/src/main/AndroidManifest.xml (hunk 1 above)
Comment (silin) [inline SwrveSDKCommon/src/main/AndroidManifest.xml:3]: Cool! Previously we had to add separate manifest merger rule to fix this `android:allowBackup="true"`

File: gradle.properties (hunk 2 above)
Comment (dominicmarmionswrve) [inline gradle.properties:25]: @Sergio-Mira Same prop in private repo needs to be updated also......

**Pull request merged**